src/Core/Severity/RealTime/Application/UseCase/FindSeverity/FindSeverity.php

```diff
@@ -41,17 +41,18 @@ public function __construct(private ReadSeverityRepositoryInterface $repository)
     }
 
     /**
+     * @param integer $severityTypeId
      * @param FindSeverityPresenterInterface $presenter
      */
-    public function __invoke(FindSeverityPresenterInterface $presenter): void
+    public function __invoke(int $severityTypeId, FindSeverityPresenterInterface $presenter): void
     {
-        $this->info('Searching for severities in the realtime');
+        $this->info('Searching for severities in the realtime', ['type' => Severity::TYPES_AS_STRING[$severityTypeId]]);
```

Comment:
i think this log could trigger a PHP Warning if the index doesnt exist in your TYPES_AS_STRING array